packages/react/ds-app-launchpad/src/ui/GitDiffViewer/common/CodeDiffViewer/style.css

```diff
@@ -0,0 +1,39 @@
+/* @canonical/generator-canonical-ds 0.0.1 */
+
+.ds.code-diff-viewer {
+  font-size: 13px;
+  .diff-table {
+    --comment-icon: url('data:image/svg+xml,<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M12 2C12.7956 2 13.5587 2.31607 14.1213 2.87868C14.6839 3.44129 15 4.20435 15 5V10C15 10.6204 14.8076 11.2256 14.4495 11.7323C14.0913 12.2389 13.5849 12.622 13 12.829V16L8.5 13H4C3.20435 13 2.44129 12.6839 1.87868 12.1213C1.31607 11.5587 1 10.7956 1 10V5C1 4.20435 1.31607 3.44129 1.87868 2.87868C2.44129 2.31607 3.20435 2 4 2H12ZM12 3.5H4C3.62712 3.50002 3.26761 3.63892 2.99158 3.88962C2.71555 4.14032 2.5428 4.48484 2.507 4.856L2.5 5V10C2.50002 10.3729 2.63892 10.7324 2.88962 11.0084C3.14032 11.2844 3.48484 11.4572 3.856 11.493L4 11.5H8.954L11.5 13.196L11.501 11.768L12.501 11.415C13.049 11.221 13.436 10.725 13.493 10.146L13.5 10V5C13.5 4.62712 13.3611 4.26761 13.1104 3.99158C12.8597 3.71555 12.5152 3.5428 12.144 3.507L12 3.5ZM7.976 8.25V9.75H4.994V8.25H7.976ZM10.981 5.25V6.75H4.994V5.25H10.981Z" fill="currentColor"/></svg>');
+    --comment-icon-width: 16px;
+
+    /* updated dynamically on window resize */
+    --table-width: 300px;
+
+    table-layout: auto;
+    border-collapse: separate;
+    border-spacing: 1px 0;
+    display: block;
+    overflow: auto;
```

Comment:
When the diff lines are horizontally scrolled, there is some strange behavior going on with the sticky line numbers. The layering becomes inconsistent.

![image](https://github.com/user-attachments/assets/84d779b8-f950-44de-a30d-710ca8d11954)

If you scroll far enough, only some line numbers are visible for some reason: 
![image](https://github.com/user-attachments/assets/8ccfcc5d-8b25-4c72-a01b-e57dc984285a)

Reply:
Thanks for checking this @jmuzina , On chrome it is working but not on Safari.